Configurations/README-design.md

```diff
@@ -4,17 +4,17 @@ Design document for the unified scheme data
 How are things connected?
 -------------------------
 
-The unified scheme takes all its data from the build.info files seen
+The unified scheme takes all its data from the `build.info` files seen
 throughout the source tree.  These files hold the minimum information
 needed to build end product files from diverse sources.  See the
-section on build.info files below.
+section on `build.info` files below.
```

Comment:
> Can you linkify this? (I don't know, I'm asking)

It would not make sense here because it talks about `build.info` files in general, not about a specific one.